lib/child_process.js

```diff
@@ -69,6 +69,11 @@ exports.fork = function fork(modulePath /* , args, options */) {
     args = arguments[pos++];
   }
 
+  if (pos < arguments.length && (arguments[pos] === undefined ||
+    arguments[pos] === null)) {
```

Comment:
This needs to be indented a bit more to align with the rest of the conditional from the line above it.

Reply:
Done.